pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -2234,15 +2234,19 @@ func LegacySaveCommandToUnstructured(cmd *dashboards.SaveDashboardCommand, names
 	finalObj.SetNamespace(namespace)
 	finalObj.SetGroupVersionKind(v0alpha1.DashboardResourceInfo.GroupVersionKind())
 
-	if cmd.FolderUID != "" {
-		meta, err := utils.MetaAccessor(&finalObj)
-		if err != nil {
-			return finalObj, err
-		}
+	meta, err := utils.MetaAccessor(&finalObj)
+	if err != nil {
+		return finalObj, err
+	}
 
+	if cmd.FolderUID != "" {
 		meta.SetFolder(cmd.FolderUID)
 	}
 
+	if cmd.Message != "" {
+		meta.SetMessage(cmd.Message)
```

Comment:
is message only for dashboards, or will it be used by other resources in the future?

Reply:
It is currently only used by dashboards, but not sure if it will be used for things in the future (cc @ryantxu ?)

Reply:
It could be used for anything -- but only used for dashboards now